assets/js/components/ClusterDetails/ChecksSelectionNew.jsx

```diff
@@ -158,6 +158,7 @@ function ChecksSelectionNew({ clusterId, cluster }) {
           {localSavingSuccess && selectedChecks.length > 0 && (
             <SuggestTriggeringChecksExecutionAfterSettingsUpdated
               clusterId={clusterId}
+              usingWanda
```

Comment:
Can we avoid referencing the raw name of the service? something like `usingNewChecksEngine` would be fine.

Also, we use the same pattern in the router. Having a shared name for all these props would allow us for a simpler `grep` in the future when we have to remove all of these switches.

Reply:
I replaced all the places where we have this pattern by this `usingWanda`, so there is not any other wording.
This variable will be removed soonish, and I preferred the wanda wording, as it makes it cristal clear what it does.
But well, I will check if you prefer the other way around

Reply:
It's ok :+1:

Reply:
https://github.com/trento-project/web/pull/1056/commits/e1b59b597e937fc9034c38de6a484fed3e21e684